app/services/twilio_api_service.rb

```diff
@@ -12,6 +9,14 @@ class NotificationService
   TWILIO_TEST_SMS_NUMBER = "+15005550006"
   TWILIO_TEST_WHATSAPP_NUMBER = "+14155238886"
 
+  class Error < StandardError
+    attr_reader :exception_message
+    def initialize(message, exception_message: nil)
+      super(message)
+      @exception_message = exception_message
+    end
+  end
+
```

Comment:
Don't need to define anything custom, plain ruby exceptions have all we need here.

```suggestion
  class Error < StandardError
  end
```